design/open/oidc-support.md

```diff
@@ -0,0 +1,22 @@
+# OIDC support in lakeFS
+
+As suggested by @arielshaqed, we started researching the option to implement OIDC support as part of lakeFS.
+
+In such a solution, the person who configures lakeFS will supply the parameters required for implementing OIDC, including the provider's domain, the client ID and secret, and a callback URL. API access will still use credentials managed internally like today.
+
+As a first step, authorization will still be managed internally by lakeFS.
+
+## Requirements
+
+
+### lakeFS admin
+- As part of the lakeFS configuration file, you can enable an integration with an OIDC provider
```

Comment:
```suggestion

- As part of the lakeFS configuration file, you can enable an integration with an OIDC provider
```

(Markdown likes a blank line after a header; it does happen to work in GFM but not in more nitpicking tools...).